### PR TITLE
Updated install.sh

### PR DESCRIPTION
**Pull request by buffet**
- Fixed a typo
- Now also checking for clang

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -19,13 +19,13 @@ installandconfigure(){
     sudo make install
 
     mkdir -p ~/.config/custard
-    cp ./examples/sxhkdrc ~/.config/custard/sxhkdrc
+    cp ./examples/sxhkdrc ~/.config/sxhkd/sxhkdrc
```
Comment (Sweets) [inline install.sh:22]: sxhkdrc should be in the custard configuration directory with the other configurations for custard. Users with sxhkd most likely had it installed prior to custard, even more likely is that they had it installed for bspwm, in which case this line would override their bspwm keybindings.

File: install.sh [hunk 2]
```diff
@@ -19,13 +19,13 @@ installandconfigure(){
...
     cp ./examples/custard.sh ~/.config/custard/rc.sh
 
     cat <<EOF>> ~/.config/custard/start.sh
 \#!/bin/bash
 echo '' > /tmp/custard.log
-sxhkd -c ~/.config/custard/sxhkd &
+sxhkd -c ~/.config/sxhkd/sxhkd &
```
Comment (Sweets) [inline install.sh:28]: Should be `~/.config/custard/sxhkdrc` (my fault, sorry)

Comment (Snuggle) [reply install.sh:28]: Could we please have this change added to `start.sh`? It's broken at the moment.

Comment (buffet) [reply install.sh:28]: @Snuggle As a temporary workaround, you can a) fix it yourself (just move the file to another path) or b) call `sxhkd` with the `-c` flag.

**Pull request closed without merging**